test/test_torch.py

```diff
@@ -6194,10 +6194,9 @@ def test_comparison_ops_check_for_scalar_overflow(self):
             torch.tensor([1 << 5], dtype=torch.uint8) < (1 << 20)
 
     def test_comparison_ops_check_for_zerodim_tensor_overflow(self):
-        with self.assertRaisesRegex(RuntimeError, 'Expected object of scalar type'):
+        with self.assertRaisesRegex(RuntimeError, 'value cannot be converted to type'):
             torch.tensor([1 << 5], dtype=torch.uint8) < torch.tensor((1 << 20), dtype=torch.int32)
```

Comment:
You don't need the (1 << 20); the parenthesis doesn't do anything.

Also, can we test a case where we are doing (zero_dim_tensor < zero_dim_tensor)?

Reply:
Changed